Src/Plugins/Tests.Security/TestData/ReviewPotentiallySensitiveFiles/ExpectedOutputs/SEC103_024.MicrosoftSerializedCertificateStoreFile_default.sarif

```diff
@@ -62,6 +62,7 @@
             }
           ],
           "fingerprints": {
+            "AssetFingerprint/v1": "[thumbprint=391550988FF0B079A0E712728F2EB95D542EF58F]",
             "ValidationFingerprint/v1": "[thumbprint=391550988FF0B079A0E712728F2EB95D542EF58F]"
```

Comment:
>391550988FF0B079A0E712728F2EB95D542EF58F [](start = 53, length = 40)

Note! This is a bug! you can't validate against a thumbprint! This thing should hold the isolated certififacte base64-encoded data. We can fix later, let's open an issue.